_posts/2017-09-30-End-to-end-IoT.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+authors: [kevin_van_den_abeele]
+title: 'Building end-to-end IoT demos with LoRa'
+image: /img/booze-5.jpg
+tags: [JavaScript, TypeScript, IoT, Internet Of Things, Arduino, Prototyping, Sensors, LoRa, Booze, Node.js, Proximus, MyThings, smart tech, smart, tech, johnny-five, electronics, modem]
+category: IoT
+comments: true
+---
+
+>To showcase end-to-end LoRa applications we built simple yet fun, real world demo applications. 
+These applications show a full end-to-end implementation of the LoRa techonology leveraging the Proximus MyThings Internet of Things platform.
+
+<p style="text-align: center;">
+  <img class="image fit" style="height: 500px; margin:0px auto;" alt="Booze-o-meter at devoxx" src="../img/end-to-end-iot/booze-devoxx.jpg">
+</p>
+
+> The Booze-o-meter V2 at Devoxx Belgium 2016.
+
+# Building end-to-end LoRa Iot Solutions 
+Building an enterprise IoT solution is challanging. Devices need to be enrolled, monitored and maintained. 
+You can roll your own network and handle all of this yourself, this however will require quite the backend system to facilitate all of this.
```

Comment:
"roll your own" with "enrolled" before might be a slight style error.
"quite the backend system" seems a bit misused here